server/tilesource/tiff.py

```diff
@@ -38,23 +38,28 @@ class TiffGirderTileSource(GirderTileSource):
     @staticmethod
     def cacheKeyFunc(args, kwargs):
         item = args[0]
-        return item.get('largeImage')
+        return item.get('largeImage') if item else None
```

Comment:
I'm not sure why you have this. If ``args[0]`` (``self``) isn't found, we've probably done something wrong, so it's fine to raise an exception. Additionally, ``None`` is a perfectly valid cache key, so there could easily be an invisible key collision.